lib/wisper/activejob/broadcaster/version.rb

```diff
@@ -1,7 +1,7 @@
 module Wisper
   module Activejob
     module Broadcaster
-      VERSION = '0.2.1'.freeze
+      VERSION = '0.3.1'.freeze
```

Comment:
When bumping a "minor" version number, the "patch" version number typically resets to zero.

Reply:
🤦  Thanks!